.test_pull_requests.json.sample

```diff
@@ -60,6 +60,7 @@
       "test_prefix": "Merge Test Results:",
       "pretest_queue_threshold": 2,
       "pretest_settings_key": "test_settings",
+      "preceding_test_settings": "test_settings",
```

Comment:
Was wondering about that, the effect is the same but the meaning is *slightly* different, I wasn't sure about overloading that setting.